Gruntfile.js

```diff
@@ -89,7 +89,8 @@ module.exports = function (grunt) {
           'js/not-selector-feature-detect.modernizr.js',
           'js/style-update-feature-detect.modernizr.js',
           'js/svg-filters-feature-detect.modernizr.js',
-          'js/ie10-viewport-bug-workaround.js'
+          'js/ie10-viewport-bug-workaround.js',
+          'js/safari-user-agent-test.js'
```

Comment:
Can we have `.modernizr.` in the filename and re-order it before the viewport bug workaround so it goes with the other modernizr tests?